code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -495,18 +495,19 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/plane_master)
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	critical = PLANE_CRITICAL_DISPLAY
 
-/// This will not work through multiz, because of a byond bug with BLEND_MULTIPLY
+///Contains all overlay (movable) lighting
+/// This still will not work through multiz, because of a byond bug with BLEND_MULTIPLY
 /// Bug report is up, waiting on a fix
-/atom/movable/screen/plane_master/o_light_visual
-	name = "Overlight light visual"
+/atom/movable/screen/plane_master/o_lighting
+	name = "Overlight Lighting"
 	documentation = "Holds overlay lighting objects, or the sort of lighting that's a well, overlay stuck to something.\
 		<br>Exists because lighting updating is really slow, and movement needs to feel smooth.\
-		<br>We draw to the game plane, and mask out space for ourselves on the lighting plane so any color we have has the chance to display."
-	plane = O_LIGHTING_VISUAL_PLANE
+		<br>Draws directly onto the lighting plate, on top of the turf lighting plane."
+	plane = O_LIGHTING_PLANE
 	appearance_flags = PLANE_MASTER|NO_CLIENT_COLOR
-	render_target = O_LIGHTING_VISUAL_RENDER_TARGET
+	render_relay_planes = list(RENDER_PLANE_LIGHTING)
+	blend_mode_override = BLEND_ADD
```

Comment:
this breaks colored overlay lights, which is why a filter is used in the first place.
I think you might be able to get a better effect using both a blend multiply and a blend add, but at that point you're just doing the filter again, and we'd also have issues with multiz unless we add a intermediary step.